sdv/multi_table/base.py

```diff
@@ -480,6 +499,10 @@ def sample(self, scale=1.0):
             total_rows += len(table)
             total_columns += len(table.columns)
 
+        for table in sampled_data:
+            if table in self._original_table_columns:
```

Comment:
For backwards compatibility, we should use `getattr` and default to an empty dict